Makefile

```diff
@@ -10,9 +10,10 @@ build:
 	sudo podman push $(IMAGE)
 
 deploy:
-	sed "s#__IMAGE__#${IMAGE}#g;s#__BM_INVENTORY_URL__#${BM_INVENTORY_URL}#g" deploy/ocp-metal-ui-template.yaml > deploy/ocp-metal-ui.yaml
-	kubectl apply -f deploy/ocp-metal-ui.yaml
-	rm deploy/ocp-metal-ui.yaml
+	mkdir -p build/deploy/
+	tools/deploy_config.sh "${BM_INVENTORY_URL}" "${IMAGE}" deploy/ocp-metal-ui-template.yaml > build/deploy/ocp-metal-ui.yaml
+	kubectl apply -f build/deploy/ocp-metal-ui.yaml
```

Comment:
Why not just deleting the generated file here?

Reply:
see above
Also this helps in debugging deploy errors.
Otherwise we do not know what the deploy file contained

Reply:
good point, doing it in `clean` is good enough